ioctl/newcmd/alias/aliasimport_test.go

```diff
@@ -12,68 +17,73 @@ import (
+			"mhs2": "io19sdfxkwegeaenvxk2kjqf98al52gm56wa2eqks",
+			"yqr":  "io1cl6rl2ev5dfa988qmgzg2x4hfazmp9vn2g66ng",
+			"mhs":  "io1tyc2yt68qx7hmxl2rhssm99jxnhrccz9sneh08",
 		},
-	}
-	client := mock_ioctlclient.NewMockClient(ctrl)
-	client.EXPECT().SelectTranslation(gomock.Any()).Return("mockTranslation", config.English).Times(4)
+	}).Times(4)
 	client.EXPECT().AliasMap().Return(map[string]string{
-		"a": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx",
-		"b": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx",
-		"c": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542s1",
-		"io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx",
+		"mhs2": "io19sdfxkwegeaenvxk2kjqf98al52gm56wa2eqks",
+		"yqr":  "io1cl6rl2ev5dfa988qmgzg2x4hfazmp9vn2g66ng",
+		"mhs":  "io1tyc2yt68qx7hmxl2rhssm99jxnhrccz9sneh08",
 	}).AnyTimes()
-	client.EXPECT().Config().Return(cfg).AnyTimes()
-
-	cmd := NewAliasImportCmd(client)
 
-	result, err := util.ExecuteCmd(cmd, `{"name":"test","address":"io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx"}`)
+	t.Run("invalid flag", func(t *testing.T) {
+		cmd := NewAliasImportCmd(client)
+		result, err := util.ExecuteCmd(cmd, "-f", "test", "test")
+		require.Error(err)
+		require.Contains(err.Error(), "invalid flag")
+		require.NotNil(result)
+	})
 
-	require.NoError(t, err)
-	require.NotNil(t, result)
+	t.Run("import alias with json format", func(t *testing.T) {
+		cmd := NewAliasImportCmd(client)
+		result, err := util.ExecuteCmd(cmd, `{"name":"test","address":"io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx"}`)
+		require.NoError(err)
+		require.NotNil(result)
+	})
 
-	result, err = util.ExecuteCmd(cmd, "-F", `{"name":"io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx","address":"io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx"}`)
-	require.NoError(t, err)
-	require.NotNil(t, result)
-
-	result, err = util.ExecuteCmd(cmd, `""{"name":"d","address":""}""`)
-	require.Error(t, err)
-	require.NotNil(t, result)
-
-	result, err = util.ExecuteCmd(cmd, "-f", "yaml", `aliases:
+	t.Run("import alias with yaml format", func(t *testing.T) {
+		cmd := NewAliasImportCmd(client)
+		result, err := util.ExecuteCmd(cmd, "-f", "yaml", `aliases:
 - name: mhs2
   address: io19sdfxkwegeaenvxk2kjqf98al52gm56wa2eqks
 - name: yqr
   address: io1cl6rl2ev5dfa988qmgzg2x4hfazmp9vn2g66ng
 - name: mhs
   address: io1tyc2yt68qx7hmxl2rhssm99jxnhrccz9sneh08`)
-	require.NoError(t, err)
-	require.NotNil(t, result)
+		require.NoError(err)
+		require.NotNil(result)
+	})
 
-	result, err = util.ExecuteCmd(cmd, "-f", "yaml", `aliases:
-name: mhs2
-  address: io19sdfxkwegeaenvxk2kjqf98al52gm56wa2eqks`)
-	require.Error(t, err)
-	require.NotNil(t, result)
+	t.Run("force import", func(t *testing.T) {
+		cmd := NewAliasImportCmd(client)
+		result, err := util.ExecuteCmd(cmd, "-F", `{"name":"io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx","address":"io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx"}`)
+		require.NoError(err)
+		require.NotNil(result)
```

Comment:
can check result?

Reply:
Seems using `force-import` will cause a syncronize issue, according to my trial, `force-import` argument parse will execute after the loop below is done.
https://github.com/iotexproject/iotex-core/blob/3f224285e5cec85162c3406f8597a390b8da5f78/ioctl/newcmd/alias/aliasimport.go#L76

Need a little help for fixing this sync issue, thx!

Reply:
If you input command with -F,  the value of `force-import` is parsed to true, will run to line81; without -F, `force-import` is parsed to false by default, will run into line76. 
-F means to force update the value of the alias, which the input alias already exists in default config file (~/.config/ioctl/default/config.default).